### ISSUE TITLE
`minishift profile set` does not report if an empty string/white space is passed as profile name.

### ISSUE THREAD

**Issue by anjannath** [minishift/minishift#1696]
```
$ minishift profile set ' '
Profile ' ' set as active profile
$ minishift profile list
-  		Does Not Exist	(Active)
- minishift	Does Not Exist
#but we can also delete the profile
$ minishift profile delete ' '
You are deleting the active profile. It will remove the VM and all related artifacts. Do you want to continue [y/N]?: y
VM for profile ' ' does not exist
Deleted:  /home/anath/.minishift/profiles/ 
Profile ' ' deleted successfully
Switching to default profile 'minishift' as the active profile.
```
we should give an error saying profile name is empty.

**Comment by amitkrout** [comment]
Is there any naming convention checks on creating profile name ?

**Comment by anjannath** [comment]
Not right now, but we could check if the profile name is an empty string or just a bunch of spaces and exit with a message.

**Comment by hferentschik** [comment]
Merged via pull request #1697
